packages/anoncreds/src/repository/AnonCredsRevocationStatusListRecord.ts

```diff
@@ -0,0 +1,46 @@
+import type { AnonCredsRevocationStatusList } from '../models'
+import type { TagsBase } from '@aries-framework/core'
+
+import { BaseRecord, utils } from '@aries-framework/core'
+
+export interface AnonCredsRevocationStatusListRecordProps {
+  id?: string
+  credentialDefinitionId: string
+  revocationStatusList: AnonCredsRevocationStatusList
+}
+
+export type DefaultAnonCredsRevocationStatusListTags = {
+  revocationRegistryDefinitionId: string
+  credentialDefinitionId: string
+  timestamp: string
+}
+
+export class AnonCredsRevocationStatusListRecord extends BaseRecord<
```

Comment:
Why do we store this object? Do we take this as the base when calling `updateRevocationStatusList`?

Reply:
Yes, we do, but it's true that we don't need a specific record for it, unless we are going to use it as a cache. However I guess such cache will be responsibility of the corresponding AnonCredsRegistry.